docs/design/ovs-pipeline.md

```diff
@@ -613,7 +613,69 @@ is required), as well as for local Pod-to-Pod traffic.
 table=70, priority=0 actions=goto_table:80
 ```
 
-### L3DecTTLTable (71)
+When the Egress feature is enabled, there will be two extra flows added in
+[L3ForwardingTable], which send the egress traffic from Pods to external network
+to [SNATTable] (rather than sending the traffic the [L2ForwardingCalcTable]
+directly). One of the flows is for egress traffic from local Pods; another
+one is for egress traffic from remote Pods, which is tunnelled to this Node to
+be SNAT'd with a SNAT IP configured on the Node. In the latter case, the flow
+also rewrites the destination MAC to the local gateway interface MAC.
+
+```text
+table=70, priority=190,ip,reg0=0x2/0xffff actions=goto_table:72
+table=70, priority=190,ip,reg0=0/0xffff actions=mod_dl_dst:e2:e5:a4:9b:1c:b1,goto_table:72
+```
+
+### SNATTable (71)
+
+This table is created only when the Egress feature is enabled. It includes flows
+to implement Egresses and select the right SNAT IPs for egress traffic from Pods
+to external network.
+
+When no Egress applies to Pods on the Node, and no SNAT IP is configured on the
+Node, [SNATTable] just has two flows. One drops egress traffic tunnelled from
+remote Nodes that does not match any SNAT IP configured on this Node, and the
+default flow that sends egress traffic from local Pods, which do not have any
+Egress applied, to [L2ForwardingCalcTable]. Such traffic will be SNAT'd with
+the default SNAT IP (by an iptables masquerade rule).
+
+```text
+table=71, priority=190,ct_state=+new+trk,ip,reg0=0/0xffff actions=drop
+table=71, priority=0 actions=goto_table:80
+```
+
+When there is an Egress applies to a Pod on the Node, a flow will be added for
```

Comment:
```suggestion
When there is an Egress applied to a Pod on the Node, a flow will be added for
```
?

Reply:
I feel "applies" works too, but changed to "applied"